client/src/schedule/EditShiftsTab.tsx

```diff
@@ -264,7 +296,7 @@ function EditShift(props: EditShiftProps) {
   useEffect(() => {
     if (shiftData) {
       setNewName(shiftData.name);
-      setNewDesc(shiftData.description);
+      // TODO: set descripion
```

Comment:
Could you add another commit reverting the change on this line and on line 321? this breaks editing the description